hack/local-up-cluster.sh

```diff
@@ -184,6 +184,39 @@ cleanup()
   exit 0
 }
 
+function createUsers {
+
+### Example token file.  Obviously you wouldn't use this in production.
+cat << EOF > /tmp/kubehack-token
+kubelet,kubelet,1
+EOF
+
+cat << EOF > /tmp/kubehack-kubeconfig
+apiVersion: v1
+kind: Config
+current-context: mycontext
+clusters:
+- cluster:
+    insecure-skip-tls-verify: true
```

Comment:
you should be able to reference the apiserver.crt instead of disabling CA verification